Description: <this is a test from branden wu>
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 sl (5.02-2) focal; urgency=medium
 .
   * [775a4f8] control: State to conform with standards 4.3.0
Author: brandenwu <raranhot@hotmail.com>
Bug-Debian: https://bugs.debian.org/744161
Bug-Debian: https://bugs.debian.org/897563

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2021-04-30

--- /dev/null
+++ sl-5.02/testing.sh
@@ -0,0 +1,2 @@
+#!/bin/bash
+echo "this is a test from Branden Wu"
